server/routes/api/helpscout.ts

```diff
@@ -0,0 +1,126 @@
+import * as express from "express";
+import {isHelpScoutIntegrationEnabled, validateHelpScoutSignature} from "../../middleware";
+import {IFormItem, IUser, User} from "../../schema";
+import bodyParser = require("body-parser");
+import * as moment from "moment-timezone";
+
+export const helpScoutRoutes = express.Router({"mergeParams": true});
+
+export type RequestWithRawBody = express.Request & { rawBody: string };
+
+helpScoutRoutes.route("/userInfo").post(
+	isHelpScoutIntegrationEnabled,
+	bodyParser.json({
+		verify: (req: RequestWithRawBody, res, buffer, encoding) => {
+			if (buffer && buffer.length) {
+				req.rawBody = buffer.toString(encoding || 'utf-8');
+			}
+		}
+	}),
+	validateHelpScoutSignature,
+	helpScoutUserInfoHandler
+);
+
+async function findUserByEmail(email: string) {
+	return User.findOne({
+		email
+	});
+}
+
+function findApplicationQuestion(applicationData: IFormItem[], questionName: string) {
+	const result = applicationData.find((item: IFormItem) => item.name === questionName);
+	return result ? result.value : "";
+}
+
+function createListBlock(title: string, content: string | string[] | Express.Multer.File | null) {
+	return `<li class="c-sb-list-item">
```

Comment:
Just happened to be scrolling through - I think this could lead to a XSS if you're not careful about the fields that are being included. You probably want to use some sort of templating library to construct this in a safe way.

Reply:
Hmmm I will look into this.  Help Scout asks you to send them basically the string HTML to display but I'm 99% certain they automatically filter what's displayed.  But never hurts to be careful on our end too

Reply:
This should be fixed now that this uses Handlebars